bin/ember-template-lint.js

```diff
@@ -232,6 +240,20 @@ function printPending(results, options) {
   }
 }
 
+function getTodoConfigFromOptions(options) {
```

Comment:
Would you mind renaming this (sorry even longer :sob: ) to `getTodoConfigFromCommandLineOptions`?

Reply:
OMG that's how I had it, but it felt so C#-y that I had to rename. I'm happy to put it back! Will harken me back to my C# days!

Reply:
Done.